### PR TITLE
TSC meeting minutes 2019-02-11

### PR DESCRIPTION
**Pull request by lgritz**

### CODE REVIEW

**Comment by lgritz** [comment]
Please let me know if I got anything wrong or left things out. And please double check the attendee list, I forgot to jot that down and guessed.

File: docs/tsc/meetings/2019-02-11.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    * Do we make an intermediate release from master before v2.0? People want a "developer preview" to be able to use the new GPU path.
+        * Patrick reluctant to make a full release.
+        * Proposal: just make a "developer preview" tag in master to designate a point we think is stable and has some helpful new features, but we don't promise any support or compatibility with the eventual v2.
+            * Down side: people will use it and expect us to support it.
+            * No consensus that this is a good idea. Mark especially doesn't like unsupported dev preview tag.
+            * But v2 will NOT be ready by SIGGRAPH 2019. In light of this, Mark thinks maybe we should have a real intermediate release.
+            * Sean: Considering how long it takes us to review things and the long v2 schedule, maybe we should reconsider what is v2 and what gets pushed to v3? (Patrick points out: it's the same ADSK people supporting the release as building "v2", only so much bandwidth available, so an intermediate release will make the full release take even longer.
+        * **Tabled** for further thought and discussion. Patrick will make a GH issue for further discussion.
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-02-11.md]: The issue is #665

Comment (lgritz) [reply docs/tsc/meetings/2019-02-11.md]: So amended.

**Comment by lgritz** [comment]
I also pushed a "template.md" file that we can use as a starting point for other meeting minutes.

**Comment by hodoulp** [comment]
@michdolan @scooperly Not having any more comments, I think that the pull request could now be merged.

**Pull request merged**